easybuild/easyconfigs/r/Racon/Racon-1.2.0-foss-2016b.eb

```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per http://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'CMakeMake'
+
+name = 'Racon'
+version = '1.2.0'
+
+homepage = 'https://github.com/isovic/racon'
+description = 'Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads'
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+# no source_urls because the tarball has to be manually generated doing
+# git clone --recursive https://github.com/isovic/racon.git racon
+# cd racon
+# git reset --hard 7295b9b  # specific to v1.2.0
+# cd ..
+# mv racon racon-1.2.0
+# tar cfvz racon-1.2.0.tar.gz racon-1.2.0
+sources = ['%(namelower)s-%(version)s.tar.gz']
```

Comment:
You should be able to do this directly, see https://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html?highlight=git%20commit#downloading-from-a-git-repository